src/Agent.Listener/CommandSettings.cs

```diff
@@ -127,6 +127,39 @@ public bool GetRunAsService()
                 defaultValue: false);
         }
 
+        public bool GetEnableAutoLogon()
+        {
+            return TestFlagOrPrompt(
+                name: Constants.Agent.CommandLine.Flags.EnableAutoLogon,
+                description: StringUtil.Loc("EnableAutoLogon"),
+                defaultValue: !Unattended);
```

Comment:
We closed on this behavior as part of the spec review. Let me confirm it with Chris and Prachi once more. I'll keep this comment active till then.

---
In reply to: [115208503](https://github.com/Microsoft/vsts-agent/pull/951#discussion_r115208503) [](ancestors = 115208503)